project/scripts/bisect.scala

```diff
@@ -86,14 +97,15 @@ object Releases:
 
     override def toString: String = version
 
-class CommitBisect(mainClass: Option[String], compilerArgs: List[String]):
+class CommitBisect(validationScriptPath: String):
   def bisect(lastGoodHash: String, fistBadHash: String): Unit =
     println(s"Starting bisecting commits $lastGoodHash..$fistBadHash\n")
-    val runOption = mainClass.map(className => s"--run $className").getOrElse("")
-    val scriptFile = Paths.get("project", "scripts", "dottyCompileBisect.sh")
-    val tempScriptFile = File.createTempFile("dottyCompileBisect", "sh").toPath
-    Files.copy(scriptFile, tempScriptFile, StandardCopyOption.REPLACE_EXISTING)
+    val bisectRunScript = s"""
+      |scalaVersion=$$(sbt "show scala3-compiler/version" | tail -1 | sed 's/\\[info\\] //')
+      |sbt "clean; scala3/publishLocal"
```

Comment:
`clean` in not enough when bisecting. You should also delete the `out` directory.

Reply:
done